File: src/luogu/p5715/Main.cpp
// https://www.luogu.com.cn/problem/P5715
#include <algorithm>
#include <cstdio>

using namespace std;
int main() {
  int arr[3];
  scanf("%i%i%i", &arr[0], &arr[1], &arr[2]);
  sort(arr, arr + 3);
  printf("%i %i %i\n", arr[0], arr[1], arr[2]);
  return 0;
}

File: src/luogu/p5715/Main.java
package luogu.p5715;

import java.util.ArrayList;
import java.util.Comparator;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        ArrayList<Integer> integers = new ArrayList<>();
        for (int i = 0; i < 3; i++) {
            integers.add(sc.nextInt());
        }
        integers.sort(Comparator.naturalOrder());
        for (Integer integer : integers) {
            System.out.print(integer + " ");
        }
        sc.close();
    }
}
